src/frontend/Komponenter/Behandling/Fanemeny/IverksetteKAVedtak.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import SendTilBeslutterFooter from '../Totrinnskontroll/SendTilBeslutterFooter';
+import { useBehandling } from '../../../App/context/BehandlingContext';
+import { AlertInfo } from '../../../Felles/Visningskomponenter/Alerts';
+import { useHentVedtak } from '../../../App/hooks/useHentVedtak';
+import { skalFerdigstilleUtenBeslutter } from '../VedtakOgBeregning/Felles/utils';
+import DataViewer from '../../../Felles/DataViewer/DataViewer';
+
+interface Props {
+    behandlingId: string;
+}
+
+const IverksetteKAVedtak: React.FC<Props> = ({ behandlingId }) => {
+    const { behandling, behandlingErRedigerbar } = useBehandling();
+    const { hentVedtak, vedtak } = useHentVedtak(behandlingId);
+
+    useEffect(() => {
+        hentVedtak();
+    }, [hentVedtak]);
+    return (
+        <DataViewer response={{ behandling, vedtak }}>
+            {({ behandling, vedtak }) => (
```

Comment:
Denne komponenten er identisk med `KorrigeringUtenBrev` ville renamet `KorrigeringUtenBrev` og gjenbrukt den. Inne i denne komponenten kan du utlede det som skal stå i `AlertInfo` på behandlingens årsak.

Reply:
Har endret navn på KorrigeringUtenBrev til BehandlingsÅrsakUtenBrev, og bruker nå denne komponenten for begge tilfeller. Teksten i alerten bestemmes avhengig av behandlingsårsak.